src/main/java/org/opensearch/ad/transport/AnomalyResultTransportAction.java

```diff
@@ -510,16 +504,16 @@ private void handleException(Exception e) {
                 return;
             } else {
                 // HC logic ends and single entity logic starts here
-                String thresholdModelID = modelPartitioner.getThresholdModelId(adID);
-                Optional<DiscoveryNode> asThresholdNode = hashRing.getOwningNodeWithSameLocalAdVersionForRealtimeAD(thresholdModelID);
-                if (!asThresholdNode.isPresent()) {
-                    listener.onFailure(new InternalFailure(adID, "Threshold model node is not available."));
+                String rcfModelID = ModelIdMapper.getRcfModelId(adID, 0);
```

Comment:
Minor: We are going to use only 1 model partition for single-flow detector. How about adding some comments here, so others can understand what 0 means?

Reply:
added